src/editorManager.js

```diff
@@ -79,7 +79,7 @@ class EditorManager {
         }
       });
 
-      frame = frame.frameElement && frame.frameElement.ownerDocument.defaultView;
+      frame = Object.getPrototypeOf(frame.parent) && frame.frameElement && frame.frameElement.ownerDocument.defaultView;
```

Comment:
Here I see a lot of repetitive code, comparing with other changes added within this PR. Maybe it's a good idea to move that part of code to the helper? Something like:
```js
export function getParentWindowSafely(currentWindow) {
  return Object.getPrototypeOf(currentWindow.parent) && currentWindow.frameElement && currentWindow.frameElement.ownerDocument.defaultView;
}
```
or simply `getParentWindow`?

Reply:
Good idea. I moved it into dom helper.